### ISSUE TITLE
Precomputed solution parameters are not in a sub-section

### ISSUE THREAD

**Issue by ktbolt** [SimVascular/svMultiPhysics#268]
### Description

There are several parameters specifying data for a precomputed solution
```
 <Use_precomputed_solution> true </Use_precomputed_solution>
 <Precomputed_solution_file_path> precomputed_velocity.vtu </Precomputed_solution_file_path>
 <Precomputed_solution_field_name> Velocity </Precomputed_solution_field_name>
```
These parameters should be in a sub-section, something like
```
<Precomputed_solution>
  <Results_file> precomputed_velocity.vtu  </Results_file>
  <Field_name> Velocity </Field_name>
</Precomputed_solution>
```

### Reproduction

See https://github.com/SimVascular/svFSIplus/blob/main/tests/cases/fluid/precomputed_dye_AD/svFSIplus.xml.

### Expected behavior

I would expect a `Precomputed_solution` sub-section to be added.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct and Contributing Guidelines

**Comment by lpapamanolis** [comment]
The DoD staff has looked into this over the last few weeks: @zasexton do you think that the precomputed parameters should remain under the general simulation parameters, or it would best fitted under the mesh parameters or the equation parameters?

**Comment by ktbolt** [comment]
@lpapamanolis What is `DoD staff` ? Department of Defense ?

**Comment by lpapamanolis** [comment]
@ktbolt Department of Debugging

**Comment by ktbolt** [comment]
@lpapamanolis It is good to have understandable comments, no cute acronyms please.

**Comment by lpapamanolis** [comment]
@ktbolt Thanks for the feedback, noted.

**Comment by zasexton** [comment]
@ktbolt @lpapamanolis yes, I will look into fixing/improving the format of the `precomputed_solution` section. In general would we still like to have this section be in the general simulation parameter section? I had some original uncertainty about this but this is how it was originally coded within a branch of `svFSI` so i decided to maintain the location.

**Comment by ktbolt** [comment]
@zasexton The other parameters used to initialize simulation state are under the `Add_mesh` section. 

Note that there are already `Initial velocities file path` and `Initial pressures file path` parameters defined under the `Add_mesh` section, and a `Simulation initialization file path` parameter under the `GeneralSimulationParameters` section. 

And now there is going to be a `Precomputed_solution` parameter. 

This is all a bit confusing.

**Comment by ktbolt** [comment]
I will add a class to `Parameters` to process the `Precomputed Solution` section.

**Comment by ktbolt** [comment]
I've added a `PrecomputedSolutionParameters` class to `Parameters` that processes the XML section under `GeneralSimulationParameters`

```
  <Precomputed_solution>
    <Use_precomputed_solution> true </Use_precomputed_solution>
    <File_path> precomputed_velocity.vtu </File_path>
    <Field_name> Velocity </Field_name>
  </Precomputed_solution>
```

I tested with `svMultiPhysics/tests/cases/fluid/precomputed_dye_AD`, worked fine.

**Comment by ktbolt** [comment]
Merged.
